openpype/lib/delivery.py

```diff
@@ -14,31 +13,24 @@ def collect_frames(files):
         Uses clique as most precise solution, used when anatomy template that
         created files is not known.
 
-        Depends that version substring starts with 'v' with any number of
-        numeric characters after.
+        Assumption is that frames are separated by '.', negative frames are not
+        allowed.
```

Comment:
I'm wondering. Should we also implement a test showing that negative frames do not work?
Like have a test that shows that e.g. `frame.-001.png` does not work? Similar to how we have a test that shows that `frame_001.png` does not parse to a frame number?

Reply:
It is already there `test_collect_frames_single_sequence_negative`